<launch>

	<!-- Machine -->
	<machine name="localhost" address="localhost" env-loader="/opt/ros/indigo/env.sh"/>
	<arg name="machine" default="localhost" />


	<node name="gap" pkg="dwl_terrain" type="default_flat_terrain" output="screen" machine="$(arg machine)">
		<remap from="topic_output" to="asus/depth_registered/points" />
		<param name="position/x" type="double" value="0." />
		<param name="position/y" type="double" value="0." />
		<param name="position/z" type="double" value="0." />
		<param name="world_frame" type="string" value="world" />
		<param name="rectangles" type="int" value="7" />
		<param name="rectangle_1/center_x" type="double" value="2." />
		<param name="rectangle_1/center_y" type="double" value="0." />
		<param name="rectangle_1/width" type="double" value="2." />
		<param name="rectangle_1/length" type="double" value="5.5" />
		<param name="rectangle_1/yaw" type="double" value="0." />
		<param name="rectangle_1/resolution" type="double" value="0.01" />
		<param name="rectangle_1/height" type="double" value="0." />

		<param name="rectangle_2/center_x" type="double" value="1.04" />
		<param name="rectangle_2/center_y" type="double" value="0." />
		<param name="rectangle_2/width" type="double" value="1." />
		<param name="rectangle_2/length" type="double" value="0.48" />
		<param name="rectangle_2/yaw" type="double" value="0." />
		<param name="rectangle_2/resolution" type="double" value="0.01" />
		<param name="rectangle_2/height" type="double" value="0.14" />

		<param name="rectangle_3/center_x" type="double" value="1.52" />
		<param name="rectangle_3/center_y" type="double" value="0." />
		<param name="rectangle_3/width" type="double" value="1." />
		<param name="rectangle_3/length" type="double" value="0.48" />
		<param name="rectangle_3/yaw" type="double" value="0." />
		<param name="rectangle_3/resolution" type="double" value="0.01" />
		<param name="rectangle_3/height" type="double" value="0.28" />

		<param name="rectangle_4/center_x" type="double" value="2." />
		<param name="rectangle_4/center_y" type="double" value="0." />
		<param name="rectangle_4/width" type="double" value="1." />
		<param name="rectangle_4/length" type="double" value="0.48" />
		<param name="rectangle_4/yaw" type="double" value="0." />
		<param name="rectangle_4/resolution" type="double" value="0.01" />
		<param name="rectangle_4/height" type="double" value="0.42" />

		<param name="rectangle_5/center_x" type="double" value="2.48" />
		<param name="rectangle_5/center_y" type="double" value="0." />
		<param name="rectangle_5/width" type="double" value="1." />
		<param name="rectangle_5/length" type="double" value="0.48" />
		<param name="rectangle_5/yaw" type="double" value="0." />
		<param name="rectangle_5/resolution" type="double" value="0.01" />
		<param name="rectangle_5/height" type="double" value="0.42" />

		<param name="rectangle_6/center_x" type="double" value="2.96" />
		<param name="rectangle_6/center_y" type="double" value="0." />
		<param name="rectangle_6/width" type="double" value="1." />
		<param name="rectangle_6/length" type="double" value="0.48" />
		<param name="rectangle_6/yaw" type="double" value="0." />
		<param name="rectangle_6/resolution" type="double" value="0.01" />
		<param name="rectangle_6/height" type="double" value="0.28" />

		<param name="rectangle_7/center_x" type="double" value="3.44" />
		<param name="rectangle_7/center_y" type="double" value="0." />
		<param name="rectangle_7/width" type="double" value="1." />
		<param name="rectangle_7/length" type="double" value="0.48" />
		<param name="rectangle_7/yaw" type="double" value="0." />
		<param name="rectangle_7/resolution" type="double" value="0.01" />
		<param name="rectangle_7/height" type="double" value="0.14" />
	</node>

</launch>
